components/CommunityEventModal.vue

```diff
@@ -326,17 +326,6 @@ export default {
 
       // If we don't have any groups, force a select.
       this.event.groups = this.event.groups ? this.event.groups : [{ id: 0 }]
```

Comment:
Maybe this line can be removed too, I didn't quite understand why it's there...

Reply:
When we first add an event, we won't have any groups.  We want to force them to choose one.  That might happen because of the order of the select options (as choose as at the top) but I see no harm in making it explicit.

Reply:
We have also seen cases in the past where events get stranded without groups due to bugs and we want to edit them to fix them up.

Reply:
I still don't really understand. It seems an event can have multiple groups (as I see the AddGroup/RemoveGroup stuff) but the UI only has a place for setting a single group?

![singlegroup](https://user-images.githubusercontent.com/31616/68590953-faa25b00-048f-11ea-8861-5a928c65b624.png)